contracts/buyback/YieldReserve.sol

```diff
@@ -245,8 +249,6 @@ contract YieldReserve is ReentrancyGuard, Ownable {
     {
         TokenData storage srcTokenData = tokenData[_srcToken];
         TokenData storage dstTokenData = tokenData[_dstToken];
-        if (!srcTokenData.srcAllowed) revert InvalidSourceToken();
```

Comment:
what is wrong with having the reverts here?

Reply:
In buyback page when users's enter tokens amounts for tokens which are either not src or dst it doesn't show the calculated value for the other token, as it reverts.

Reply:
Should we handle it on the UI, like if it reverts- show an unsupported pair error?
This is not a big issue, but I am reluctant about this change because we can use this function at other places where the revert might be expected. For ex, we can use this function internally and forget about adding the additional reverts. 
But if such situation are unlikely then we can go ahead with this change.

Reply:
Nice point. @YashP16 wdyt?

Reply:
I think we can't have the revert statements here because, 
current deployed code doesnot allow to compute the reverse token logic i.e how much tokenA is required to buy tokenB as we will have to add all destination tokens as source tokens.